### ISSUE TITLE
 sagemaker 

### ISSUE THREAD

**Issue by almfahd41** [biplobsd/OneClickRun#81]
hello 

is there a copy of this notebook can run on free  https://studiolab.sagemaker.aws/ notbooks ?

thanks

**Comment by biplobsd** [comment]
Some of features are only works on Google Colab platform. So maybe not working as works in Colab. But you can test and share your report in here.

**Comment by almfahd41** [comment]
> Some of features are only works on Google Colab platform. So maybe not working as works in Colab. But you can test and share your report in here.

yes all my tries failed

**Comment by biplobsd** [comment]
I don't have access to sagemaker. It is saying request free account..

**Comment by almfahd41** [comment]
> @almfahd41, I don't have access to sagemaker. It is saying request free account..

yes just sing up and you will get it winthin 24 houre ,  it is like google colab you will get 4 hours of gpu amd 8 hour of cpu

**Comment by almfahd41** [comment]
@biplobsd did you got access to sagemaker ? 

or i can make a new account for you

**Comment by biplobsd** [comment]
@almfahd41 they did not mail yet 😃
